include/iso15118/io/connection_ssl.hpp

```diff
@@ -14,11 +14,13 @@ namespace iso15118::io {
 struct SSLContext;
 class ConnectionSSL : public IConnection {
 public:
-    ConnectionSSL(PollManager&, const std::string& interface_name, const config::SSLConfig&);
+    ConnectionSSL(PollManager&, const std::string&, const config::SSLConfig&);
```

Comment:
Why did you remove the parameter name here?

Reply:
Since the other parameters also have no name.

Reply:
That makes sense. I got a little bit confused here